### PR TITLE
Bot deletes messages outside command channel and alerts user

### PR DESCRIPTION
**Pull request by jamesjamail**
Completes "Bot should inform people commands in general channel aren't recognized"

Also made minor tweaks to !help message

Resolves: #18

### CODE REVIEW

File: client.js [hunk 1]
```diff
@@ -13,19 +13,19 @@ const helpMsg = '\n\n' +
     '\n' +
     '***COMMANDS:***\n' +
     '!help' +
-    '> Displays available commands.\n' +
+    '   →   Displays available commands.\n' +
```
Comment (rm-you) [inline client.js:16]: Ah, ">" is specifically for quoting, it is special discord formatting. I BELIEVE it should look better than a unicode arrow? But I'll look at both and see what I think shortly.

File: client.js [hunk 2]
```diff
@@ -51,8 +51,9 @@ bot.on('ready', () => {
 
 bot.on('message', function (message) {
     // console.log(message);
-    // ignore bots or messages from any text channel that isn't the Command Channel
-    if (message.author.bot || (message.channel.type === 'text' && message.channel.id !== COMMAND_CHANNEL)){
```
Comment (rm-you) [inline client.js:55]: Now that we don't short-circuit on messages in other channels, won't it hit the `else` below on messages without "!" and then spam channels when people talk?

File: client.js [hunk 3]
```diff
@@ -159,6 +160,11 @@ bot.on('message', function (message) {
                 message.author.send('Sorry, I didn\'t recognized that command.  Please check your syntax and try again. ' + helpMsg);
                 break;
         }
+        // message the general channel that commands are not recognized in this channel if command detected:
+        if (!message.author.bot && message.channel.type === 'text' && message.channel.id !== COMMAND_CHANNEL){
+            bot.channels.cache.get(COMMAND_CHANNEL).send(`Hi <@${message.author.id}>, I received your command but deleted it from the \`general\` channel to keep things tidy.  In the future, please use this channel instead or send me a direct message.  Thanks!`)
+            message.delete();
+        }
     }
     else {
```
Comment (rm-you) [inline client.js:169]: I would expect this else block to activate on normal chat in #general now?

File: client.js [hunk 4]
```diff
@@ -159,6 +160,11 @@ bot.on('message', function (message) {
                 message.author.send('Sorry, I didn\'t recognized that command.  Please check your syntax and try again. ' + helpMsg);
                 break;
         }
+        // message the general channel that commands are not recognized in this channel if command detected:
+        if (!message.author.bot && message.channel.type === 'text' && message.channel.id !== COMMAND_CHANNEL){
+            bot.channels.cache.get(COMMAND_CHANNEL).send(`Hi <@${message.author.id}>, I received your command but deleted it from the \`general\` channel to keep things tidy.  In the future, please use this channel instead or send me a direct message.  Thanks!`)
```
Comment (rm-you) [inline client.js:165]: Do we want to spam the command channel with this? Or should we just PM the user directly? I also don't necessarily like the idea of *publically* shaming users for making a mistake. Deleting the message has the advantage of hiding the mistake quickly, but then we announce about it loudly... :/

File: client.js (hunk 1 above)
Comment (jamesjamail) [reply client.js:16]: Oh, I didn't notice.  Let me know.

File: client.js (hunk 2 above)
Comment (jamesjamail) [reply client.js:55]: You're right, fixed.

File: client.js (hunk 3 above)
Comment (jamesjamail) [reply client.js]: This is the same as your comment above I believe? If so, fixed.

File: client.js (hunk 4 above)
Comment (jamesjamail) [reply client.js:165]: I liked how it works when you are @'ed from the channel you are suppose to use.  If you think there's a shame factor though we can make it a PM.

Comment (rm-you) [reply client.js:165]: Yeah, I suppose it's fine. It's more like an FYI, I suppose. There's technically other cool stuff we could do, like "click the checkmark reaction to acknowledge this message" and then delete that too... but lots of extra work for no real reason :D

File: client.js (hunk 1 above)
Comment (rm-you) [reply client.js:16]: Yeah, IMO it looks better with the quoting, if you want to change it back 👍

**Comment by jamesjamail** [comment]
> I'm good with this once you swap back to the line-quote `>`

It's ready captain.

File: client.js (hunk 1 above)
Comment (jamesjamail) [reply client.js:16]: Done!

File: client.js [hunk 5]
```diff
@@ -1,7 +1,7 @@
 const Discord = require('discord.js');
 const logger = require('winston');
 const settings = require('./settings.json');
-const { helpMsg, welcomeMsg } = require('./messages');
+const { helpMsg, welcomeMsg } = require('./messages')
```
Comment (rm-you) [inline client.js:4]: RIP semicolon

**Pull request merged**